lib/Stats.js

```diff
@@ -867,6 +873,9 @@ class Stats {
 			if (module.built) {
 				colors.green(" [built]");
 			}
+			if (module.assets && module.assets.length) {
+				colors.magenta(` [${module.assets.length} asset]`);
```

Comment:
for `module.assets.length > 1` it should be asset**s** instead of asset

Reply:
@sokra 
Thank you for the review, I have changed it.

I thinks it's better to extract `module.assets.length` into a variable such as `const assetsSize = module.assets.lengt`, but for keeping the same behavior with other sources(`module.warnings, module.errors`), I directly used `module.assets.length`.